web/src/message_edit.js

```diff
@@ -753,7 +760,9 @@ export function start_inline_topic_edit($recipient_row) {
         }),
     );
     message_lists.current.show_edit_topic_on_recipient_row($recipient_row, $form);
-    $form.on("keydown", handle_inline_topic_edit_keydown);
+    $form.on("keydown", (e) => {
+        handle_inline_topic_edit_keydown(e, $recipient_row);
+    });
```

Comment:
I guess a risk of passing `$recipient_row` is that if some sort of rerender replaced the recipient row, this handler might be affected. But I think any such rerender would probably also replace `$form`, so this seems safe enough.